model/autosklearn/dffml_model_autosklearn/config.py

```diff
@@ -1,17 +1,23 @@
 import abc
 
+import pathlib
 import importlib
-from typing import Optional
+from typing import Optional, AsyncIterator
 
 from dffml.base import config, field
+from dffml.model.accuracy import Accuracy
+from dffml import Feature, Features, Sources, Record
 from dffml.model.model import ModelContext, ModelNotTrained
-from dffml import Feature, Features, Sources
 
 
 @config
 class AutoSklearnConfig:
     predict: Feature = field("Label or the value to be predicted")
     features: Features = field("Features on which to train the model")
+    directory: pathlib.Path = field(
+        "Directory where model should be saved",
+        default=pathlib.Path("~", ".cache", "dffml", "autosklearn"),
```

Comment:
We have gone through and removed all the default directories for models. So let's not add a default here.

```suggestion
```